cmd/oras/internal/option/packer.go

```diff
@@ -99,7 +99,8 @@ func (opts *Packer) LoadManifestAnnotations() (annotations map[string]map[string
 	}
 	if opts.AnnotationFilePath != "" {
 		if err = decodeJSON(opts.AnnotationFilePath, &annotations); err != nil {
-			return nil, err
+			return nil, fmt.Errorf("error loading manifest annotations from file %s: %w, 
+				Refer to the document at https://oras.land/docs/how_to_guides/manifest_annotations ", opts.AnnotationFilePath, err)
```

Comment:
```suggestion
			return nil, fmt.Errorf("error loading manifest annotations file %s: %w", opts.AnnotationFilePath, err)
```

Reply:
I don't think we want to get in the habit of putting document urls in code. I think we are better off creating a troubleshooting guide that contains error messages and their resolution.

Reply:
ok @TerryHowe , I will have to wait and see what @qweeah has to say about it but I get your point. Let's wait for @qweeah 's review.

Reply:
@TerryHowe Adding link to the doc is a specific request from @toddysm. Code in the option package is directly related to parsing the command line input, so I am okay with adding a link here.